index.js

```diff
@@ -0,0 +1,97 @@
+let clickHandler = function(event){
+    if(event.target.parentNode.className == "done"){
+        event.target.src = event.target.src.endsWith("unticked.png")? "ticked.jpeg" : "unticked.png";
+    }
+    if(event.target.parentNode.className=="remove"){
+        let ul = event.target.parentNode.parentNode.parentNode;
+        ul.removeChild(event.target.parentNode.parentNode);
+        if(ul.children.length===0){
+            let li = document.createElement('LI');
+            li.appendChild(document.createTextNode("you have no tasks"));
+            ul.appendChild(li);
+        }
+    }
+}
+
+let updateInput = function(event){
```

Comment:
In this case, it might be better to save a reference to the input element, and get the value straight from it, instead of having an event handler